src/main/java/io/jenkins/plugins/analysis/warnings/PVSStudio.java

```diff
@@ -0,0 +1,75 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import edu.hm.hafner.analysis.IssueParser;
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.jenkinsci.Symbol;
+import hudson.Extension;
+
+import io.jenkins.plugins.analysis.core.model.IconLabelProvider;
+import io.jenkins.plugins.analysis.core.model.ReportScanningTool;
+import io.jenkins.plugins.analysis.core.model.StaticAnalysisLabelProvider;
+
+import edu.hm.hafner.analysis.parser.pvsstudio.PVSStudioParser;
+
+/**
+ * Provides a parser and customized messages for PVS-Studio static analyzer.
+ *
+ * @author PVS-Studio Team
+ */
+public class PVSStudio extends ReportScanningTool {
```

Comment:
Can you please also add a small test in the style of [ParsersITest.java](https://github.com/jenkinsci/warnings-ng-plugin/blob/master/src/test/java/io/jenkins/plugins/analysis/warnings/ParsersITest.java#L114) that counts the warnings of a single file? You can reuse the file of the analysis-model test.